staging/src/k8s.io/component-helpers/scheduling/corev1/nodeaffinity/nodeaffinity.go

```diff
@@ -225,7 +225,7 @@ func nodeSelectorRequirementsAsSelector(nsm []v1.NodeSelectorRequirement, path *
 		case v1.NodeSelectorOpLt:
 			op = selection.LessThan
 		default:
-			errs = append(errs, field.NotSupported(p.Child("operator"), expr.Operator, nil))
+			errs = append(errs, field.NotSupported[string](p.Child("operator"), expr.Operator, nil))
```

Comment:
We need to specify the type param explicitly because of passing nil here.

Reply:
uhm... we should list the supported operators instead

Reply:
Agree. But I think we can leave that on a follow-up PR.
What do you think?